src/az-button.js

```diff
@@ -64,15 +63,14 @@ export class AzButton extends DelegateFocusMixin(LitElement) {
     else {
       let openAzOffCanvasMenu = new Event('open-az-offcanvas-menu');
       document.querySelector(this.target).dispatchEvent(openAzOffCanvasMenu);
-      console.log(this);
     }
   }
 
-  render() {	
+  render() {
```

Comment:
the below code, basically has two options, either render a `button`, or an  `a` element depending on if there is a link attribute.